ViAn/Project/project.cpp

```diff
@@ -57,6 +65,7 @@ ID Project::add_video_project(VideoProject *vid_proj){
 void Project::remove_video_project(VideoProject* vid_proj){
     auto it = std::find(m_videos.begin(), m_videos.end(), vid_proj);
     if (it == m_videos.end()) return;
+    delete *it;
```

Comment:
Do we always want to delete the video project here. Not just remove it from the project?